lms/templates/instructor/instructor_dashboard_2/add_coupon_modal.html

```diff
@@ -3,7 +3,8 @@
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
 %>
-<section id="add-coupon-modal" class="modal" role="dialog" tabindex="-1" aria-label="${_('Add Coupon Code')}">
+<section id="add-coupon-modal" class="modal" role="dialog" tabindex="-1" aria-labelledby="header-coupon-code">
```

Comment:
Labelled by the child heading `id`. These sections aren't dynamically generated on this page, so there currently isn't a way to make this programatic. Each `section` here should be labelled by its child heading.